parser.js

```diff
@@ -0,0 +1,27 @@
+module.exports.templateParser=(templateString,templateData)=>{
+    var customTagsRegex = /\(\((#[a-z]+ )?[a-z]+.[a-z]*\)\)/g;
```

Comment:
Could you explain this regex?

Reply:
https://regexr.com/4gnhc

Reply:
What is the purpose of the first capture group, `(#[a-z]+ )`?

Can you make this match a greater variety of placeholders such as `((x))`, `((TEST))`, `((manufacturer-product-code))`, `((2nd-name))`?

Reply:
yeah i guess the first capture group doesn't add value
```suggestion
    var customTagsRegex = /\(\(([a-zA-Z0-9-]+ )?.[a-zA-Z0-9-]*\)\)/g;
```
I updated the regexr link